rand-core/README.md

```diff
@@ -0,0 +1,21 @@
+rand-core
+====
+
+Core functionality for the [rand] library.
+
+This crate contains the core trait, `RngCore`, and some extension traits. This
+should be sufficient for libraries publishing an `RngCore` implementation,
```

Comment:
This reads a like a suggestion, but we prefer RNG implementations to only depend on this crate, right? Can you reword a little?